hubblestack/files/hubblestack_nova/win_reg.py

```diff
@@ -195,8 +195,8 @@ def _find_option_value_in_reg(reg_hive, reg_key, reg_value):
         for sid in key_list:
             if len(sid) <= 15 or '_Classes' in sid:
                 continue
-            reg_key = reg_key.replace('<SID>', sid)
```

Comment:
this loop only works for first sid. Once "<SID>" is replaced by the first sid, it is never replaced again in further loops.